expr/expr.go

```diff
@@ -100,6 +102,8 @@ func compileNative(node ast.Expression) (NativeEvaluator, error) {
 			return compileIn(lhsFunc, rhsFunc)
 		case "=", "!=":
 			return compileCompareEquality(lhsFunc, rhsFunc, n.Operator)
+		case "=~":
```

Comment:
Are we considering `!~` too at some point?

Reply:
I hadn't really thought about it.  We can always write `! (foo =~ pattern)`, but a new operator would be easy enough to add...